aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -672,6 +735,14 @@ module AptosFramework::Stake {
         // Otherwise, we can move directly to inactive.
         if (is_current_validator(pool_address)) {
             Coin::merge<TestCoin>(&mut stake_pool.pending_inactive, unlocked_stake);
+
+            // Update the amount of stake being withdrawn from the active validator set's total voting power.
```

Comment:
if it's already a pending_inactive validator, the voting power is being counted twice

Reply:
Updated